react/StyleGuideProvider/StyleGuideProvider.js

```diff
@@ -7,6 +7,10 @@ import classnames from 'classnames';
 import Helmet from 'react-helmet';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
+if (typeof window !== 'undefined') {
+  require('smoothscroll-polyfill').polyfill();
```

Comment:
[ca-profile-ui/scroll](https://github.com/SEEK-Jobs/ca-myprofile-ui/blob/master/app/modules/scroll/index.js) is our scroll module. It's not profile specific though. Could be pulled out. Only thing is it's configurable, so we'd have to agree on settings.